### ISSUE TITLE
Unable to start activity - getSdkComponent returns null after app was killed

### ISSUE THREAD

**Issue by freszu** [onfido/onfido-android-sdk#66]
#### What was the expected behaviour?
App shouldn't crash when coming back to front after it was killed
#### What happened instead?
App is crashing with Null pointer exception
#### Version info:

- `onfido-capture-sdk` version: 4.8.1
- Android API version(s): all

#### Steps to reproduce:
This was spotted in our error tracking tool , by now we have 110 crashes caused by this on various devices for OnfidoActivity and OnfidoCaptureActivity. 

This is quite simple to reproduce and is caused by the false assumption in the code that SdkController will be always initialised before starting onfido activities and this is not the case when device is under memory pressure in which case after minimizing the app it can be killed with it's state being saved in onSaveInstanceState and restored when coming back to the app, if we are on any of onfido activities this will cause the NPE when retrieving the sdk component.

To reproduce and simulate this :
1. Open app
2. Go to any onfido screen
3. Minimize the app 
4. Close the app with ```adb shell am kill com.sample.package```
5. Reopen app it will try to restore it's state but will fail because of this bug
#### Additional LogCat output:
```
kotlin.c: null
    at com.onfido.android.sdk.capture.common.SdkController.getSdkComponent
    at com.onfido.android.sdk.capture.common.SdkController.getSdkComponent$default
    at com.onfido.android.sdk.capture.ui.BaseActivity.attachBaseContext
    at android.app.Activity.attach(Activity.java:7059)
    at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2911)
    at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3086)
    at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:78)
    at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:108)
    at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:68)
    at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1816)
    at android.os.Handler.dispatchMessage(Handler.java:106)
    at android.os.Looper.loop(Looper.java:193)
    at android.app.ActivityThread.main(ActivityThread.java:6718)
    at java.lang.reflect.Method.invoke(Method.java)
    at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
    at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
java.lang.RuntimeException: Unable to start activity ComponentInfo{com.xxx/com.onfido.android.sdk.capture.ui.OnfidoActivity}: kotlin.c
    at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2951)
    at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3086)
    at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:78)
    at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:108)
    at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:68)
    at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1816)
    at android.os.Handler.dispatchMessage(Handler.java:106)
    at android.os.Looper.loop(Looper.java:193)
    at android.app.ActivityThread.main(ActivityThread.java:6718)
    at java.lang.reflect.Method.invoke(Method.java)
    at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
    at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
```
#### Other useful info:
_screenshots, videos, etc._

**Comment by sergiomarquesmoura** [comment]
Duplicate of #64. Fixed in `4.9.0`
